apps/artemis-web/libs/next-csrf.js

```diff
@@ -0,0 +1,2243 @@
+Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
Is this boilerplate or custom code? Should it be reviewed or is it just an included library?

Reply:
no it is just an included library.. I just made some minor modifications to their development branch. The master branch was buggy